### PR TITLE
added blog post for typelevel sustainability program

### PR DESCRIPTION
**Pull request by kailuowang**
added link to program page

### CODE REVIEW

File: posts/2019-4-24-typelevel-sustainability-program-announcement.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+We are excited to announce the Typelevel sustainability program.
+The ultimate goal for this program is to provide ways for the user community to ensure the long-term sustainability of the development and maintenance of some Typelevel libraries.
+Currently, these libraries are maintained in their contributors spare time. This arrangement has worked so far but we want to firmly secure their long term sustainability with an institution dedicated to supporting the maintenance of these mission critical libraries.
```
Comment (larsrh) [inline posts/2019-4-24-typelevel-sustainability-program-announcement.md]:
```suggestion
Currently, these libraries are maintained in their contributors' spare time. This arrangement has worked so far but we want to firmly secure their long term sustainability with an institution dedicated to supporting the maintenance of these mission critical libraries.
```

File: posts/2019-4-24-typelevel-sustainability-program-announcement.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Based on our [Cats ecosystem community survey](https://typelevel.org/blog/2019/01/30/cats-ecosystem-community-survey-results.html), roughly 70% of the users will gain more confidence
+in the future of the ecosystem if there are compensated maintainers. We believe that our pure FP
+Scala ecosystem should have an institution supporting it, somewhat like the Scala Center
+supporting the language, and Lightbend supporting language as well as ecosystem libs. 
+
+There are also some concerns, 3% of users suggested paid maintainers will reduce their confidence, and 27.1% are not sure one way or another. Given these numbers, we believe that the gain of confidence will outweigh the concerns especially if we design our paid maintainership program with deliberations to address them. The worries we received fall into four categories:
```
Comment (larsrh) [inline posts/2019-4-24-typelevel-sustainability-program-announcement.md]: `27.1` is too specific. Just say "the remainder".

File: posts/2019-4-24-typelevel-sustainability-program-announcement.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+* Revamp guidance documentation for contributors
+* Revamp Documentations Navigation
+* More tutorial/example documentation
+* Complete cats-tagless' migration away from scala-meta
+* Merge in typelevel/algebra
+* A community build for the ecosystem
+
+Our initial fundraising goal is $150,000. Among other things, achieving this goal will allow us to have a dedicated half-time (20 hr/week) maintainer for at least 2019. Why half time? We want to start with a committed maintainer to bring some certainties for our projects, and yet we are not sure how much support we will be getting from the community. Hence a half-time maintainer for the year is a minimum viable solution for the program. 
+
+Please consider talking to your employee about supporting the OSS libraries they are using. Any amount, either $5 per month from a one-person start-up or $5000 per month from a billion dollar corporation, will bring us closer to our goal.
```
Comment (larsrh) [inline posts/2019-4-24-typelevel-sustainability-program-announcement.md]:
```suggestion
Please consider talking to your employer about supporting the OSS libraries they are using. Any amount, either $5 per month from a one-person start-up or $5000 per month from a billion dollar corporation, will bring us closer to our goal.
```

File: posts/2019-4-24-typelevel-sustainability-program-announcement.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* Sponsoring events such as free training, conferences, by providing the venue, food, etc. 
+* Donations of training/support services or coupons that we can then exchange for monetary contributions.
+
+For individual developers, another way to support us monetarily is to help our fundraising effort by: 
+
+* spreading the news about our fundraising campaign
+* mentioning and linking to our donation page when you write a blog post or give a talk about one of these libraries
+* advocating for the sustainable OSS development
+
+Please don't hesitate to reach out with questions. Our contact Email is sponsor-contact@typelevel.org
```
Comment (larsrh) [inline posts/2019-4-24-typelevel-sustainability-program-announcement.md]:
```suggestion
Please don't hesitate to reach out with questions. Our contact address is sponsor-contact@typelevel.org.
```

**Comment by kailuowang** [comment]
@larsrh feedback addressed. Also I am curious if the netlify preview for PR still works. I tried
https://deploy-preview-252--typelevel.netlify.com/blog/2019/04/24/typelevel-sustainability-program-announcement.html
but got not found.

**Comment by larsrh** [comment]
No, Netlify is disabled completely.

**Pull request merged**